ibm_was/datadog_checks/ibm_was/__about__.py

```diff
@@ -1,4 +1,4 @@
 # (C) Datadog, Inc. 2018-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-__version__ = '1.12.0'
+__version__ = '1.13.0'
```

Comment:
```suggestion
__version__ = '1.12.1'
```

Reply:
how did this happen?

Reply:
The changes included a duplicate security change, so it increased the version to 1.13.0 rather than just 1.12.1 that it should have for the small fix.